_includes/markdown/CSS.md

```diff
@@ -323,7 +280,7 @@ Articles worth reading:
 
 <h2 id="responsive-websites">Responsive websites {% include Util/top %}</h2>
 
-We build our websites mobile first. We do not rely on `respond.js` as it does not work well in certain environments. Instead, we leverage the benefits of Sass to manipulate our breakpoints.
+We build our websites mobile first. We do not rely on JavaScript libraries such as `respond.js` as it does not work well in certain environments. Instead, we leverage a natural, movile-first build process and allow sites gracefully degrade.
```

Comment:
Typo in *movile*

Reply:
Also, thoughts on touching on small screen first vs. mobile first here?

Reply:
@tlovett1 What do you think about either just swapping the language here from "mobile" to "small screen" _or_ swapping the language _and_ addressing mobile vs. small screen in another section of the site?

Reply:
Small screen works for me!